### PR TITLE
{lib}[intel/2018b] mkl_fft v1.0.14 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by arielzn**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/mkl_fft/mkl_fft-1.0.14-intel-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    ('Python', '3.6.6'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+source_urls = ['https://github.com/IntelPython/mkl_fft/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['644ed4d845b741f80c6576c94bdbd14d18fce21e6cc725676e979b7f953154a8']
```
Comment (lexming) [inline easybuild/easyconfigs/m/mkl_fft/mkl_fft-1.0.14-intel-2018b-Python-3.6.6.eb]: Please move this part between `toolchain` and `dependencies`. It is not a strict error but we try to keep some common structure between the easyconfigs.

File: easybuild/easyconfigs/m/mkl_fft/mkl_fft-1.0.14-intel-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+source_urls = ['https://github.com/IntelPython/mkl_fft/archive']
```
Comment (lexming) [inline easybuild/easyconfigs/m/mkl_fft/mkl_fft-1.0.14-intel-2018b-Python-3.6.6.eb]: The `source_urls` can be simplified by using `GITHUB_SOURCE`
```suggestion
github_account = 'IntelPython'
source_urls = [GITHUB_SOURCE]
```

File: easybuild/easyconfigs/m/mkl_fft/mkl_fft-1.0.14-intel-2018b-Python-3.6.6.eb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+sources = ['v%(version)s.tar.gz']
+checksums = ['644ed4d845b741f80c6576c94bdbd14d18fce21e6cc725676e979b7f953154a8']
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s'],
+}
```
Comment (lexming) [inline easybuild/easyconfigs/m/mkl_fft/mkl_fft-1.0.14-intel-2018b-Python-3.6.6.eb]: This `sanity_check_paths` are default with `PythonPackage`. They can be removed.

**Comment by arielzn** [comment]
Test report by @arielzn
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login1.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0721902049261755d01f6805327777e9 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b15515c828141ab5fff99fcc74724110 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @arielzn !

**Pull request merged**